diffkemp/diffkemp.py

```diff
@@ -468,7 +470,11 @@ def compare(args):
             old_fun_desc.mod.clean_module()
             new_fun_desc.mod.clean_module()
             LlvmModule.clean_all()
-
+        if output_dir is not None:
+            dump_compare_output(snapshot_dir_old=args.snapshot_dir_old, 
+                                snapshot_dir_new=args.snapshot_dir_new, 
+                                result=result,graph=result_graph,
```

Comment:
Doesn't the `result.graph` already have the graph, making it unnecessary to pass it in another argument?

Reply:
Thanks, it seems that you are correct.